client/post-editor/editor-sharing/accordion.jsx

```diff
@@ -82,18 +83,23 @@ const EditorSharingAccordion = React.createClass( {
 	},
 
 	hideSharing: function() {
-		return this.props.site &&
-			this.props.site.jetpack &&
-			! this.props.site.isModuleActive( 'publicize' ) &&
-			! this.props.site.isModuleActive( 'sharedaddy' ) &&
-			! this.props.site.isModuleActive( 'likes' );
+		return ( this.props.site &&
+				this.props.site.jetpack &&
+				! this.props.site.isModuleActive( 'publicize' ) &&
+				! this.props.site.isModuleActive( 'sharedaddy' ) &&
+				! this.props.site.isModuleActive( 'likes' ) ) ||
+			( this.props.site &&
+				this.props.site.jetpack &&
+				this.props.postType === 'page' &&
+				! this.props.site.isModuleActive( 'sharedaddy' ) &&
+				! this.props.site.isModuleActive( 'likes' ) );
```

Comment:
Looking again, I see this component already has an `isPublicizeEnabled` prop which will do nicely here.  Another alternative which I think I like better than the array + `every` nonsense:

```js
return (
    ! site.isModuleActive( 'sharedaddy' ) &&
    ! site.isModuleActive( 'likes' ) &&
    ( ! isPublicizeEnabled || ! site.isModuleActive( 'publicize' ) )
);
```